receiver/snowflakereceiver/testdata/config.yaml

```diff
@@ -0,0 +1,14 @@
+snowflake:
+  # required settings
+  username: snowflakeuser
+  password: securepassword
+  account: bigbusinessaccount
+  warehouse: metricWarehouse
+  # optional settings (i.e. have defaults)
+  collection_interval: 18m
+  metrics:
+    snowflake.database.bytes_scanned.avg:
+      enabled: true
+    snowflake.database.bytes_deketed.avg:
+      enabled: false
+  role: customMonitoringRole 
```

Comment:
These config files in testdata are usually used in the tests like here. Feel free to add a test for this config in a follow-up PR

Reply:
sure no problem, probably should have to begin with